README.md

```diff
@@ -0,0 +1,5 @@
+# mflowd
+A go daemon that takes metrics from metrics-flow and exposes them to prometheus
```

Comment:
Would be good to explain what is going on here. I think the pubsub messages need to be of a specific format? And you have teh various aggregation optionns.